example/index.js

```diff
@@ -91,7 +92,7 @@ const params = {
 
 	cameraProjection: 'Perspective',
 
-	backgroundType: 'Gradient',
+	backgroundType: 'Environment',
```

Comment:
I'd prefer not to change any of the settings in these examples - we should just add the denoiser.

Reply:
No worries. My fear is how bad the denoiser treats the gradient background compared to envMaps. But I'll revert it